docs/3.x.x/getting-started/quick-start.md

```diff
@@ -1,285 +1,281 @@
 
-::: note
-See the [CLI documentation](../cli/CLI.md#strapi-generateapi) for more information on how to do it the hacker way.
+   The **Name** Field
+
+   Click on the **"+ Add New Field"** button.
+
+   Next we will click on the **String** field.
+
+   You should be under the **BASE SETTINGS** tab, in the **Name** field, type **"Name"**. This will be the name of the category.
+
+   Now, click on the **ADVANCED SETTINGS** tab, check **Required field** and **Unique field**. This field is required for each category and is not optional. Also, we want each category to have a unique name, so we check the **Unique field** box.
+
+   And then click the **Continue** button.
+
+3. We are now ready to add the second field, the **Relation** field for the **Many to Many relation** setting between Categories and Restaurants.
+
+   The **Relation** Field
+
+   Click the **"+ Add New Field"** button.
+
+   From here, click the **Relation** field.
+
+   This will bring you to the **Add New Relation** screen. We will change two elements on this page for the **Relation** field.
+
+   The first element to change is **Permission (Users-Permissions)** to **Restaurant** (on the right).
+
+   The second element to change is to click the **Many to Many** icon (in the middle). It should now read, **"Categories has and belongs to many Restaurants"**.
+
+   Then click the **Continue** button.
+
+4. Lastly, we will save this Content Type. Click the **Save** button.
+
+5. Wait for Strapi to restart.
+
+Verify in the left menu under **CONTENT TYPES**, that you see **Categories**, **Restaurants** and **Users**.
+
+We are now ready to [manage and add data to the Content Types](#_4-manage-and-add-data-to-content-type) we just created.
+
+Next, click on the **Strapi** logo (top left corner) to return the main administration panel.
+
+::: warning NOTE
+See the [CLI documentation](../cli/CLI.md#strapi-generateapi) for more information on how to add Content Types the hacker way.
 :::
 
 ### Files structure
 
-A new directory has been created in the `./api` folder of your application which contains all the needed stuff for your `Post` Content Type: routes, controllers, services and models. Take a look at the [API structure documentation](../concepts/concepts.md#files-structure) for more informations.
+A new directory has been created in the `./api` folder of your application which contains all the needed files related to your `restaurant` and `category` Content Types: routes, controllers, services and models. Take a look at the [API structure documentation](../concepts/concepts.md#files-structure) for more information.
+
+---
+
+## 4. Manage and add data to Content Type
+
+After creating [the Content Types](#_3-create-a-content-type), we now need to manage and add data/content to the new Content Types.
```

Comment:
Add gif Onboarding https://drive.google.com/drive/folders/19h91Z5hUXTzsEhyZfhFiS9EQVSlorlGc

Reply:
We have now the videos for each section.